user-manager-service/src/main/java/com/crmfoodestablishment/usermanager/crud/dto/UpdateUserRequestDTO.java

```diff
@@ -0,0 +1,46 @@
+package com.crmfoodestablishment.usermanager.crud.dto;
+
+import com.crmfoodestablishment.usermanager.crud.entity.Role;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import jakarta.validation.constraints.*;
```

Comment:
here too wildcard import. And look at all the code there are still such wildcard imports.